ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/Time.scala

```diff
@@ -1,25 +0,0 @@
-// Copyright (c) 2020 The DAML Authors. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package com.daml.ledger.api.testtool.tests
-
-import java.time.Duration
-
-import com.daml.ledger.api.testtool.infrastructure.Allocation._
-import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTestSuite}
-
-final class Time(session: LedgerSession) extends LedgerTestSuite(session) {
-  test("PassTime", "Advancing time should return the new time", allocate(NoParties)) {
-    case Participants(Participant(ledger)) =>
-      for {
-        t1 <- ledger.time()
-        _ <- ledger.passTime(Duration.ofSeconds(1))
```

Comment:
Wanna remove the `passTime` method too?

Reply:
Good catch, thanks.

Addressed by https://github.com/digital-asset/daml/commit/2912a2948770af23b22545944698d951e2e044be